specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-03-01-preview/Capabilities.json

```diff
@@ -0,0 +1,562 @@
+              "$ref": "#/definitions/CapabilitiesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforPostgreSQL/flexibleServers/{serverName}/capabilities": {
+      "get": {
+        "tags": [
+          "LocationBasedCapabilities"
+        ],
+        "operationId": "ServerCapabilities_List",
+        "x-ms-examples": {
+          "ServerCapabilitiesList": {
+            "$ref": "./examples/ServerCapabilities.json"
+          }
+        },
+        "description": "Get capabilities at specified location in a given subscription.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../types/common-types.json#/parameters/ServerNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CapabilitiesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "StorageTierCapability": {
+      "type": "object",
+      "description": "Represents capability of a storage tier",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name to represent Storage tier capability",
+          "readOnly": true
+        },
+        "iops": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Supported IOPS for this storage tier",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../types/common-types.json#/definitions/CapabilityBase"
+        }
+      ]
+    },
+    "StorageMbCapability": {
+      "type": "object",
+      "properties": {
+        "supportedIops": {
+          "type": "integer",
+          "format": "int32",
```

Comment:
>int32 [](http://example.com/codeflow?start=21&length=5)

isnt this a breaking change ?

Reply:
We got this reviewed for breaking change